discovery-provider/src/api/v1/playlists.py

```diff
@@ -428,3 +430,27 @@ def get(self, version):
         )
         playlists = get_full_trending_playlists(request, args, strategy)
         return success_response(playlists)
+
+
+playlist_stream_bp = Blueprint("playlist_stream", __name__)
+
+
+@playlist_stream_bp.route("/v1/playlists/<string:playlist_id>/stream", methods=["GET"])
```

Comment:
Is it possible to follow the pattern of using flask restx like the other endpoints? This allows us to get types generated and usage in the sdk.

Like:
https://github.com/AudiusProject/audius-protocol/pull/4255/files#diff-884e66e1cc33e95a72ee4ed8d4d190c149175fb659f0b38348ad0e12eccaeeabR350

Or maybe that breaks because of the response type not being JSON? (I figure that pattern/lib should be able to handle this though)

Reply:
The response being non-JSON breaks this, yes